src/tests/test_connection.py

```diff
@@ -576,3 +577,216 @@ def test_feedback_when_switching_connection_without_alias(ip_empty, tmp_empty, c
 
     captured = capsys.readouterr()
     assert "Switching to connection duckdb://" == captured.out.replace("\n", "")
+
+
+@pytest.fixture
+def conn_sqlalchemy():
```

Comment:
We can rename this to `conn_sqlalchemy_duck`

Reply:
done